diff --git a/configure.ac b/configure.ac
index 3d4c4f0..5eff32b 100644
--- a/configure.ac
+++ b/configure.ac
@@ -12,7 +12,7 @@ dnl checks for programs
 AC_PROG_MAKE_SET
 AC_PROG_CC
 AC_PROG_INSTALL
-LT_INIT
+LT_INIT([disable-shared])
 AC_PROG_LIBTOOL
 
 AC_CONFIG_MACRO_DIR([m4])
diff --git a/git-version-gen b/git-version-gen
index 42cf3d2..edda463 100755
--- a/git-version-gen
+++ b/git-version-gen
@@ -98,6 +98,7 @@ elif
     && case $v in
 	 [0-9]*) ;;
 	 v[0-9]*) ;;
+	 fairwaves/[0-9]*) v=`echo "$v" | sed 's|fairwaves/||'` ;;
 	 *) (exit 1) ;;
        esac
 then
